### DISCUSSION TITLE
How loud is the recommended relay board from the Aliexpress?

### DISCUSSION THREAD

**Discussion by onnimonni** [Lucifer06/RemoteGPIO#40] [Q&A] [answered]
I'm interested in automating roof ventilation and floor warming in my van. Has someone checked how loud sounds can be heard when the relay starts or stops?

The scenario I have in mind is that the relay will wake me up in the middle of the night when automation kicks in 😅.

If someone who has this setup in use would be able to use decibel meter that would be amazing 🙇.

**Comment by onnimonni** [comment, marked as answer]
I bought it and one can definitely hear the click but it isn't too loud especially when it's in a enclosure.

The relay clicks don't bother me at all even though I'm sleeping 40cm above the relay.

**Comment by Lucifer06** [comment]
Agreed there are not louder than the ones in the CerboGX
